src/routes/RocketsPage.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRockets, toggleReserveRocket } from '../redux/rockets/rocketSlice';
+import RocketCard from '../components/RocketCard';
+
+const RocketsPage = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const status = useSelector((state) => state.rockets.status);
+  const reservedRockets = useSelector((state) => state.rockets.reservedRockets);
+
+  useEffect(() => {
+    if (status !== 'idle') {
+      dispatch(fetchRockets());
+    }
+  }, [status, dispatch]);
```

Comment:
- [ ] It would be better to not fetch the data every time you reload the page in order to increase the performance as you did for the missions page 
<img width="1437" alt="Screenshot 2023-05-18 at 23 06 01" src="https://github.com/christianonoh/space-travel/assets/61790587/71193768-41a7-499e-8d3f-50eac569be19">